src/libraries/System.Text.RegularExpressions/src/System.Text.RegularExpressions.csproj

```diff
@@ -59,7 +59,7 @@
     <Compile Include="System\Text\RegularExpressions\Symbolic\BooleanClassifier.cs" />
     <Compile Include="System\Text\RegularExpressions\Symbolic\CharKind.cs" />
     <Compile Include="System\Text\RegularExpressions\Symbolic\DfaMatchingState.cs" />
-    <Compile Include="System\Text\RegularExpressions\Symbolic\PartitionClassifier.cs" />
+    <Compile Include="System\Text\RegularExpressions\Symbolic\MintermClassifier.cs" />
```

Comment:
non sequitur - I noticed UnicodeCategoryPredicateName is also dead. or maybe should be #if debug.